src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx

```diff
@@ -28,7 +28,7 @@ export const ConfigureBeaconNode: React.FunctionComponent<IConfigureBNProps> = (
     const [networkIndex, setNetworkIndex] = useState(0);
     const defaults = getDefaultsForClient(props.clientName);
 
-    const defaultChainDataDir = path.join(getConfig(remote.app).storage.dataDir);
+    const defaultChainDataDir = path.join(getConfig(remote.app).storage.home) + "/beacon";
```

Comment:
I don't think we should store chain data in ~/beacon. It's wrong on so many levels 😄 
I think according to xdg, you should use `path.join(getConfig(remote.app).storage.dataDir, "beacon");`

https://www.electronjs.org/docs/api/app#appgetpathname